src/renderer/fonts/caching_shaper.rs

```diff
@@ -134,21 +142,32 @@ impl CachingShaper {
 
             // Add guifont fallback list
             if let Some(options) = &self.options {
-                font_fallback_keys.extend(
-                    options
-                        .fallback_list
-                        .iter()
-                        .map(|font_name| font_name.into()),
-                );
+                font_fallback_keys.extend(options.fallback_list.iter().map(|font_name| FontKey {
+                    italic: options.italic,
+                    bold: options.bold,
+                    font_selection: font_name.into(),
+                }));
             }
             // Add default font
-            font_fallback_keys.push(FontKey::Default);
+            font_fallback_keys.push(FontKey {
+                italic: false,
```

Comment:
Should default and skia fallback use options.italic and options.bold? I think that would maybe help the system pick a font which matches a little better

Reply:
I have no idea, I just figured "default" would mean no extra options like that set?

Reply:
I think in this case its better to set them from the options. Default here means fall back to this when the current glyph isn't present in the configured fallback list.

Reply:
I think this is the only remaining work for this PR. Thanks so much for taking this on!

Reply:
> I think in this case its better to set them from the options. Default here means fall back to this when the current glyph isn't present in the configured fallback list.

Pushed the change, look good?

> I think this is the only remaining work for this PR. Thanks so much for taking this on!

No problem, thank you for the detailed info on how to do this! Next up is https://github.com/Kethku/neovide/issues/333#issuecomment-869234020 I think, hopefully I can get that working too.